internal/api/runme/parser/v1/parser.proto

```diff
@@ -4,6 +4,11 @@ package runme.parser.v1;
 
 option go_package = "github.com/stateful/runme/internal/gen/proto/go/runme/parser/v1;parserv1";
 
+message Runme {
```

Comment:
@pastuxso how about FrontmatterRunme? Move down above Frontmatter?

Reply:
I like it, let me change it.